src/main/java/frc/robot/Constants.java

```diff
@@ -137,12 +137,31 @@ public AimModes previous() {
 
   // AUTONOMOUS Constants
 
-  // Path json files
-  public static final String taxiPath = "paths/TaxiOut.wpilib.json";
-  public static final String oneBallPath = "paths/TaxiOutFromFender.wpilib.json";
-  public static final String twoBallPath[] = {
-    "paths/TaxiOutToGrabBall.wpilib.json", "paths/GoBackIntoFender.wpilib.json"
-  };
+  public enum Auton {
+    TAXI("paths/TaxiOut.wpilib.json"),
+    ONEBALL("paths/TaxiOutFromFender.wpilib.json"),
+    TWOBALL("paths/TaxiOutToGrabBall.wpilib.json", "paths/GoBackIntoFender.wpilib.json"),
+    THREEBALL,
+    FOURBALL,
+    FIVEBALL,
+    TEST(
+        "paths/FlippedVectorTest1.wpilib.json",
+        "paths/FlippedVectorTest2.wpilib.json"); // change according to what path you want to test
```

Comment:
This is definitely a spot where spotless makes things less readable. You can disable it for chucks of code like this:
```
// spotless: off
<code you want to manual format>
// spotless: on 
```